azure-pipelines.yml

```diff
@@ -27,34 +27,6 @@ resources:
       name: xamarin/yaml-templates
       endpoint: xamarin
 
-trigger:
```

Comment:
please remove these changes

Reply:
Odd I havn't touched that file... Ok will check and update standby.